libs/ha/ha_common_libs_k8s.py

```diff
@@ -1701,3 +1701,33 @@ def poll_to_get_resource_status(self, exp_sts, rsc, rsc_id,
         if status != exp_sts:
             return False
         return True
+
+    def failover_pod(self, pod_obj, pod_yaml, failover_node):
+        """
+        Helper function to delete, recreate and failover pod
+        :param pod_obj: Object of the pod
+        :param pod_yaml: Dict containing pod name and yaml file to be used for recreation
```

Comment:
better name it pod_dict then for better understanding

Reply:
This dict will have pod_name: yaml file path, so named it as pod_yaml.